### PR TITLE
Alpha/card-update

### PR DESCRIPTION
**Pull request by iondrimba**
**CHANGELOG** :memo:

* Updated Card compoment
* Updated media-box component
* Updated Card documentation

captura de tela 2017-04-27 as 3 58 37 pm

captura de tela 2017-04-27 as 3 58 59 pm

### CODE REVIEW

File: src/scss/components/_icon.scss [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
-  display: inline-block;
-  margin: 2px 20px 0;
+  display: flex;
   vertical-align: top;
+  font-style: italic;
+  line-height: 1;
+  color: get-color((color: blackNeutral, opacity: .4));
+  align-items: center;
 
   img {
```
Comment (eduardojmatos) [inline src/scss/components/_icon.scss]: @iondrimba I think that is a great opportunity to change `img` to a class. Scorpion project has a svg inside `icon` block already. What do you think?

File: src/scss/components/_picture.scss [hunk 2]
```diff
@@ -39,6 +39,12 @@
     }
   }
 
+  &--media {
+    max-width: 30px;
+    max-height: 30px;
```
Comment (eduardojmatos) [inline src/scss/components/_picture.scss]: @iondrimba we had a bug using max-width on our new page of public request. Flexbox calculates this using some strange logic and the image inside stay with a wrong width, like the screenshot bellow:

screen shot 2017-04-27 at 11 52 17 pm

To resolve this, we can apply a `min-width: 30px` instead `max-width`. Or we can just add `width: 100%` with this `max-width`.

File: src/scss/generic/_heading.scss [hunk 3]
```diff
@@ -151,6 +151,16 @@ h6,
     font-size: 12px;
   }
 
+  &--mini {
```
Comment (eduardojmatos) [inline src/scss/generic/_heading.scss]: This modifier seems to be on a wrong location. Is it necessary to be here or can we move this to inside of card scope?

File: src/scss/components/_media_box.scss [hunk 4]
```diff
@@ -15,35 +15,38 @@
   color: get-color((color: blackNeutral, opacity: .6));
 
   &__title {
-    color: get-color(punch);
+
+    .caption-title--mini {
```
Comment (eduardojmatos) [inline src/scss/components/_media_box.scss]: My point about this selector: https://github.com/getninjas/gaiden/pull/99/files#diff-08fca2c2524b8f14547a7595bce358f7R154

File: src/scss/components/_picture.scss (hunk 2 above)
Comment (iondrimba) [inline src/scss/components/_picture.scss]: @eduardojmatos  ok, I will add 100% to this modifier 👍

File: src/scss/generic/_heading.scss (hunk 3 above)
Comment (iondrimba) [inline src/scss/generic/_heading.scss]: @eduardojmatos Yes, I was in doubt of where the right place should be. Here I can use it as caption-title--mini. Inside the card scope I will have to test what the implementation would be. Do you prefer inside the card?

File: src/scss/components/_icon.scss (hunk 1 above)
Comment (iondrimba) [inline src/scss/components/_icon.scss]: @eduardojmatos You are right, will do that 👍

File: src/scss/components/_picture.scss (hunk 2 above)
Comment (iondrimba) [inline src/scss/components/_picture.scss]: @eduardojmatos added you suggestion of 100% to the modifier :+1:

File: src/scss/components/_icon.scss [hunk 5]
```diff
@@ -1,10 +1,14 @@
 .icon {
-  display: inline-block;
-  margin: 2px 20px 0;
+  display: flex;
   vertical-align: top;
```
Comment (MarcoBrunoBR) [inline src/scss/components/_icon.scss]: @iondrimba if we are using `` `flex``` is this` `vertical-align``` necessary?

**Comment by iondrimba** [comment]
@MarcoBrunoBR Removed the unnecessary vertical-align, nice tip!

**Comment by iondrimba** [comment]
[(5) Update da versão 2 (design) do Gaiden](https://trello.com/c/k2izpoRz/406-5-update-da-versao-2-design-do-gaiden)

**Comment by iondrimba** [comment]
@MarcoBrunoBR Any changes still needed?

**Pull request merged**